printer_mods/sgr33n/300mm-x-extruder-endcap/.metadata.yml

```diff
@@ -0,0 +1,30 @@
+---
+# The title of your mod. Choose a short and descriptive name
+title: 300mm X-Extruder Endcap
+# A short description of your mod. Try to describe your mod with 1-2 short sentences
+description: This mod makes stabler and prettier the use of a 300mm profile for the X-Extruder.
+# Mod version: Set this to 1 when you are submitting a new mod. When you make changes to your mod, you may increment this number
+mod_version: 1
+# You may add your discord username here so people can contact you about your mod
+discord_username: sgr33n
+# The CAD files for your mod. Please provide them as a list.
+# Note that each mod must include at least one CAD file.
+cad:
+- CAD/300mm_X-Extruder_Endcap.step
+# The parts of your mod, exported and oriented ready to be sliced. Please provide them as a list.
+# Note that each mod must include at least one STL or OBJ file.
+stl:
+- STL/x_extruder_endcap_left.stl
+- STL/x_extruder_endcap_right.stl
+# The image files provided with your mod. These may be instructions you use in your README.md file, or other additional pictures.
+# **Note that the first image specified here will be used as cover image on mods.vorondesign.com**
+# All paths are relative to this .metadata.yml file.
+images:
+- Images/300mm-X-Extruder-Endcap-X-Carriage.jpg
```

Comment:
It looks like `gallery-1.jpg` and `gallery-2.jpg` are missing from this list.

Reply:
Added to .metafile.yml, thanks :)